src/wolfgame.py

```diff
@@ -3383,23 +3371,27 @@ def transition_day(cli, gameid=0):
     # 6 = rearranging victim list (ensure bodyguard/harlot messages plays),
     #     fixing killers dict priority again (in case step 4 or 5 added to it)
     # Actually killing off the victims happens in transition_day_resolve
+    # We set the variables here first; listeners should mutate, not replace
+    # We don't need to use User containers here, as these don't persist long enough
+    # This removes the burden of having to clear them at the end or should an error happen
```

Comment:
Comment block seems unnecessary, and getting rid of the ability to replace data dict entries if they're containers is something I probably think we should do in general (doesn't need to be part of this PR though).

That said, you missed a bunch of places using revt.data["blah"] below which was made under the assumption that replacement is possible -- can replaces those with the vars directly.